### ISSUE TITLE
Add VEBA deployment methods to the main page of the website

### ISSUE THREAD

**Issue by rguske** [vmware-samples/vcenter-event-broker-appliance#517]
**Is your feature request related to a problem? Please describe.**
It's not immediately and easy recognizable which deployment methods for VEBA exists, when visiting the website.

**Describe the solution you'd like**
It will be beneficial to put it on the main page.

Example:
![veba_deployment_methods](https://user-images.githubusercontent.com/31652019/127216984-2e0c6d3b-a777-498a-9bea-dd117ddff9ab.png)

**Comment by rguske** [comment]
@pksrc Is it possible to use the same frames/squares you've used directly at the top of the main page?

**Comment by rguske** [comment]
@lamw @embano1 @pksrc 
What do you think? 😁 
![image](https://user-images.githubusercontent.com/31652019/131255524-ab0af2be-e0a9-4231-ad34-7766943f1486.png)

**Comment by embano1** [comment]
Cool! Would reword the K8s part to:

`Deploy ... to an existing Kubernetes Knative installation with Helm`
